circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_inner.cpp

```diff
@@ -76,23 +76,6 @@ void validate_this_private_call_hash(DummyComposer& composer,
         CircuitErrorCode::PRIVATE_KERNEL__CALCULATED_PRIVATE_CALL_HASH_AND_PROVIDED_PRIVATE_CALL_HASH_MISMATCH);
 };
 
-void validate_this_private_call_stack(DummyComposer& composer, PrivateKernelInputsInner<NT> const& private_inputs)
-{
-    const auto& stack = private_inputs.private_call.call_stack_item.public_inputs.private_call_stack;
-    const auto& preimages = private_inputs.private_call.private_call_stack_preimages;
-    for (size_t i = 0; i < stack.size(); ++i) {
-        const auto& hash = stack[i];
-        const auto& preimage = preimages[i];
-
-        // Note: this assumes it's computationally infeasible to have `0` as a valid call_stack_item_hash.
-        // Assumes `hash == 0` means "this stack item is empty".
-        const auto calculated_hash = hash == 0 ? 0 : preimage.hash();
-        composer.do_assert(hash == calculated_hash,
-                           format("private_call_stack[", i, "] = ", hash, "; does not reconcile"),
-                           CircuitErrorCode::PRIVATE_KERNEL__PRIVATE_CALL_STACK_ITEM_HASH_MISMATCH);
-    }
-};
```

Comment:
Why remove this? I think we need to do this once the pedersen + generator works in noir?

Reply:
The function `common_validate_call_stack` does this exact same thing so this function was redundant (unless I missed something). So I removed this.